### PR TITLE
Updated README with directions for tagged release, updates package-lock.json

### PR DESCRIPTION
**Pull request by jermnelson**
Updates package-lock.json with the latest version. Directions in README now include step for package-lock.json in the tagged release section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,9 +184,10 @@ $ aws ecs update-service --service sinopia-homepage --region us-west-2 --cluster
 ```
 
 ## Release Management
-The steps to create a tagged release of the Sinopia's Linked Data Editor are as follows:
+The steps to create a major tagged release of the Sinopia's Linked Data Editor are as follows:
 
 1. Update the version in `package.json`
+1. Remove and regenerate `package-lock.json`
```
Comment (mjgiarlo) [inline README.md]: Would `npm i` suffice for this step? Not sure we need to remove the file.

File: README.md [hunk 2]
```diff
@@ -184,9 +184,10 @@ $ aws ecs update-service --service sinopia-homepage --region us-west-2 --cluster
 ```
 
 ## Release Management
-The steps to create a tagged release of the Sinopia's Linked Data Editor are as follows:
+The steps to create a major tagged release of the Sinopia's Linked Data Editor are as follows:
```
Comment (mjgiarlo) [inline README.md]: Wouldn't we want to follow these steps for minor and patch releases as well?

File: README.md (hunk 1 above)
Comment (jermnelson) [reply README.md]: Done, updated in latest commit.

File: README.md (hunk 2 above)
Comment (jermnelson) [reply README.md]: Yes, I agree! Having the same steps for major, minor, and patch makes it less confusing. Remove *major* from latest commit.

**Pull request merged**